This does not set a default options as suggested in configure.ac,
but overrides the user-provided setting.

Index: configure
--- configure.orig
+++ configure
@@ -14416,7 +14416,6 @@ fi
 
 fi
 
-enable_opt="yes"
 
 
 if test "x$enable_opt" = xyes; then :
